Shared/Samples/Set map image layer sublayer visibility/SetMapImageLayerSublayerVisibilityView.swift

```diff
@@ -0,0 +1,104 @@
+// Copyright 2025 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct SetMapImageLayerSublayerVisibilityView: View {
+    /// The tracking status for the loading operation.
+    @State private var isLoading = false
+    
+    /// The error shown in the error alert.
+    @State private var error: Error?
+    
+    @State private var map: Map = {
+        let map = Map()
+        return map
+    }()
+    
+    @State private var mapImageLayer: ArcGISMapImageLayer = {
+        let imageLayer = ArcGISMapImageLayer(url: .arcGISMapImageLayerSample)
+        return imageLayer
+    }()
```

Comment:
This can be just changed to,
```suggestion
    /// The map image layer.
    @State private var mapImageLayer = ArcGISMapImageLayer(url: .arcGISMapImageLayerSample)
```